### PR TITLE
Samples: rewrite simple app tutorial.

### PR DESCRIPTION
**Pull request by tswast**
- Add additional region tags to support tutorial rewrite.
- Use more direct map for printing results.
- Use more relevant query from public datasets.

### CODE REVIEW

**Comment by tswast** [comment]
/cc @lukesneeringer @jmdobry 

(I don't have write access here, so can't add you as reviewers)

File: samples/queries.js [hunk 1]
```diff
@@ -20,41 +20,42 @@ function printResult(rows) {
...
+  // [END bigquery_simple_app_client]
+
+  // [START bigquery_simple_app_query]
+  // The SQL query to run
+  const sqlQuery = `SELECT
+    CONCAT(
+      'https://stackoverflow.com/questions/',
+      CAST(id as STRING)) as url,
+    view_count
+    FROM \`bigquery-public-data.stackoverflow.posts_questions\`
```
Comment (stephenplusplus) [inline samples/queries.js]: The auto-linter (`npm run prettier`) is getting a bit tripped up over this line. Escaping seems the most natural solution, but feel free to suggest an alternative.

Comment (tswast) [reply samples/queries.js]: Escaping works for me.

**Comment by tswast** [comment]
The docs use a commit hash to freeze the sample to a specific version, so this should be safe to merge even though I changed some of the region tags.

**Pull request merged**